tox.ini

```diff
@@ -34,7 +34,6 @@ setenv =
     MALLET_HOME={env:MALLET_HOME:}
 
 commands =
-    python -c "from gensim.models.word2vec import FAST_VERSION; print(FAST_VERSION)"
```

Comment:
Why? This is kind of "sanity check".

Reply:
Since we are dropping support for pure-Python implementations, this check fails. It is required that we first run `python setup.py build_ext --inplace` before running this command.